src/components/WorkspaceDialog.vue

```diff
@@ -15,12 +14,17 @@
         width="500"
       >
         <template v-slot:activator="{ on }">
+          <!-- See
+            https://github.com/vuetifyjs/vuetify/issues/4482#issuecomment-476689473
+            for why the `dark` prop is tied to the disabled state of the button.
+          -->
           <v-btn
             id="add-workspace"
+            :disabled="userInfo == null"
```

Comment:
```suggestion
            :disabled="userInfo === null"
```

Reply:
Thanks. I'm surprised the linter didn't catch this...